webui/react/src/utils/storage.ts

```diff
@@ -84,3 +84,9 @@ export class Storage {
     this.store.setItem(path, item);
   }
 }
+
+export const globalStorage = new Storage({ basePath: 'global', store: window.localStorage });
+
+// DISCUSS we could set this up through react contexts but then all usage of APIs
+// would have to be moved to follow suit.
+export const STORAGE_KEY_AUTH = 'auth-token';
```

Comment:
**suggestion:** maybe having class-based global storage that extends off of `Storage`, with the two specific sets of uses baked in (auth-token and server-address). ie. `setAuthToken`, `getAuthToken`, `setServerAddress`, `getServerAddress`.